source/tutorial_page.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <script src="https://cdn.lr-in-prod.com/LogRocket.min.js" crossorigin="anonymous"></script>
+    <script>window.LogRocket && window.LogRocket.init('umcvob/texascodeem');</script>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title name="tab-title">Pomodoro's tutorial page by Texas Codem</title>
+
+
+    <link rel="stylesheet" href="./styles/tutorial_page.css">
+
+    <link rel="icon" type="image/png" href="./favicon.ico">
+
+
+</head>
+
+<body>
+
```

Comment:
A bit of divitis, items could perhaps be refactored into figure/figcaption elements